src/world_interface/real_world_interface.cpp

```diff
@@ -91,13 +90,23 @@ void initMotors() {
 		// TODO: set limit switch limits
 	}
 
+	// initialize motor objects and add them to map
+	addMotorMapping(motorid_t::frontLeftWheel, false);
+	addMotorMapping(motorid_t::frontRightWheel, false);
+	addMotorMapping(motorid_t::rearLeftWheel, false);
+	addMotorMapping(motorid_t::rearRightWheel, false);
+
 	for (motorid_t motor : pidMotors) {
 		can::deviceserial_t serial = motorSerialIDMap.at(motor);
 		pidcoef_t pid = motorPIDMap.at(motor);
 		can::motor::setMotorPIDConstants(serial, pid.kP, pid.kI, pid.kD);
+
+		// initialize motor objects and add them to map
+		addMotorMapping(motor, true);
```

Comment:
I'd say move this to the enc motor and pot motor for loops, since I'm not 100% sure that every positional motor is also a pid motor. So to be clear, the two loops would each have an `addMotorMapping()` call.